docs/src/main/asciidoc/quartz.adoc

```diff
@@ -124,6 +124,58 @@ public class TaskBean {
 3. Create a new `Task` with the current start time.
 4. Persist the task in database using https://quarkus.io/guides/hibernate-orm-panache[Panache].
 
+=== Scheduling Jobs Programatically
+
+It is also possible to leverage the Quartz API directly.
+You can inject the underlying `org.quartz.Scheduler` in any bean:
+
+[source,java]
+----
+package org.acme.quartz;
+
+@ApplicationScoped
+public class TaskBean {
```

Comment:
We should probably update [Quartz QS](https://github.com/quarkusio/quarkus-quickstarts/tree/development/quartz-quickstart)  with this example

Reply:
I don't think so. This is more like a corner case, ie. something we don't want to push...